### ISSUE TITLE
Missing file/folder "files\syncCto_backups\…".

### ISSUE THREAD

**Issue by fritzmg** [menatwork/syncCto#246]
When you have a fresh Contao (and syncCto) installation and want to create a backup for the first time, the backup cannot be created, because the folder `files\syncCto_backups\database` (for example) does not exist yet.

In past versions this folder was automatically created, but now it seems you have to do it manually?

**Comment by stefanheimes** [comment]
You are right, there is a run once which should create it. I will take a look.

**Comment by fritzmg** [comment]
Yeah, I just confirmed it in a fresh installation. When installing syncCto (via Composer), the folder does not get created.

**Comment by fritzmg** [comment]
This also poses a security threat if you create these folders without the `.htaccess` files in it.

**Comment by m-vo** [comment]
+1 adding an ``.htaccess`` file is important, especially if folder listing is somehow enabled.
